gettext-template/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM elementary/docker:unstable
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install -y devscripts equivs meson git python3-git libxml2-utils appstream policykit-1 sudo
+RUN apt-get -yqq update && apt-get install -yqq devscripts equivs meson git python3-git libxml2-utils appstream policykit-1 sudo tzdata
```

Comment:
`-qq` will also `--assume-yes`, so there's no need to specify `-y`

related manpage entry: https://linux.die.net/man/8/apt-get

Reply:
done